docs/self-hosted-appcircle/self-hosted-runner/runner-vm-setup.md

```diff
@@ -512,6 +512,68 @@ tart clone macOS_240514 vm01
 
 In docker terminology, `vm01` and `vm02` will be our docker images. We will configure them separately, persist our changes and then create containers to execute build pipelines. On every build, fresh containers will be used for both runners.
 
+### Configure Runner VM Resources
+
+You can adjust the resources limits for Runner VMs based on your needs.
+
+:::info 
+By default, our runner images are configured with an 8GB memory limit and 4 CPU cores.
+:::
+
+:::caution 
+Total allocated resources should not exceed host machine's physical capacity for optimal performance.
+
+For example:
+
+| Host Memory Size | Runner Memory Config. |
+|------------------|-----------------------|
+| 8GB              | 1 x VM with 8GB       |
+| 16GB             | 1 x VM with 16GB      |
+| 16GB             | 2 x VM with 8GB       |
+| 32GB             | 2 x VM with 16GB      |
+:::
+
+#### Set Memory Limit
+
+To configure the memory limit for a VM, run the following commands:
+
+   ```bash
+   cd $HOME/.tart/vms/
+   ```
+   ```bash
+   tart set <vm_name> --memory <size_in_mb>
+   ```
+
+   Replace `vm_name` with your VM's name and `size_in_mb` with the desired memory size in MB. E.g., 
+   > `tart set vm01 --memory 8192`.
+
+#### Set CPU Limit
+
+To configure the number of CPU cores for a VM, run the following commands:
+
+   ```bash
+   cd $HOME/.tart/vms/
+   ```
+   ```bash
+   tart set <vm_name> --cpu <count>
+   ```
+
+   Replace `vm_name` with your VM's name and `count` with the desired number of CPU cores. E.g., 
+   > `tart set vm01 --cpu 4`.
+
```

Comment:
Can you add a tip box for checking the total CPU count on the host? Users may define more CPUs than the hosts total CPUs.

`sysctl -n hw.ncpu`

Also mention that if they will create only one VM, use all cores. But if there will be 2 VMs, calculate half of the total CPU count and use that value.